wtr_backend_project/src/config/index.ts

```diff
@@ -0,0 +1,31 @@
+import dotenv from "dotenv";
+
+class Config  {
+    public readonly database: {
+        host?: string;
+        port?: number;
+        username?: string;
+        password?: string;
```

Comment:
Esse database seria usado onde? Porque o `data-source.ts` usa as env vars diretamente.  
Acho que seria interessante colocar o `data-source` nessa pasta config (e ajustar onde for usado). E pra não ter esse uso das vars em dois locais, eu vejo como melhor opção setar aqui como static na classe Config e chamar esse atributo da classe lá no data-source.

Reply:
pra gerenciar a conexão com o banco, a ideia é injetar ele nos repositories e usar a api do typeorm pra interagir com o banco, coloquei pra pegar pelo .env pq não achei uma maneira legal de colocar ele pra pegar as configs do config quando rodando as migrations

Reply:
Entendi. Beleza então, acho que não tem problema assim.